src/Sulu/Bundle/EventLogBundle/Domain/Event/DomainEvent.php

```diff
@@ -130,4 +130,9 @@ public function getResourceSecurityType(): ?string
     {
         return null;
     }
+
+    public function getResourceSecurityObjectId(): ?string
```

Comment:
the `SecurityCondition` has a `getObjectType` and a `getObjectId` method. would be nice to have a consistent prefix in the `DomainEvent` and `EventRecord` too. i guess i have not thought about this enough when i implemented the `getResourceSecurityType` method 🙈 

there are two ways to have a consistent prefix:

- rename `resourceSecurityType` to `resourceSecurityObjectType`
- rename `resourceSecurityObjectId` to `resourceSecurityId` 

i dont think it will make a big difference in the future, so both ways are okay for me. but i would like to be consistent 🙂